### ISSUE TITLE
Error installing package in Node 14.x

### ISSUE THREAD

**Issue by dylankelly** [raineorshine/npm-check-updates#1207]
- [ x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [ x] I am using the latest version of `npm-check-updates`
- [ x] I am using `node >= 14.14`

---

## Steps to Reproduce

- Use Node version 14 lts/fermium (14.20.1)
- Run `npx npm-check-updates` (also true when installing globally `npm install -g npm-check-updates`)

Dependencies:

```json

```

Steps:

## Current Behaviour

Error unable to find file in dependency - @npmcli/installed-package-contents/index.js 

```
❯ nvm install lts/fermium
v14.20.1 is already installed.
Now using node v14.20.1 (npm v6.14.17)
❯ npx npm-check-updates
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /Users/dylankelly/.npm/_npx/70756/lib/node_modules/npm-check-updates/node_modules/@npmcli/installed-package-contents/index.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/dylankelly/.npm/_npx/70756/lib/node_modules/npm-check-updates/node_modules/@npmcli/installed-package-contents/index.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

## Expected Behaviour

npm-check-updates installs correctly

**Comment by dylankelly** [comment]
This seems to be an issue with @npmcli/installed-package-contents - have reported this here  https://github.com/npm/installed-package-contents/issues/24. Leaving this issue for visibility.

**Comment by meganwalker-ibm** [comment]
I think it might arguably be a "breaking-change-in-a-patch-release" problem with pacote - installing pacote@15.0.0 works fine on npm@6; but pactoe@15.0.1 does not. In that patch version they pulled in a new major version of installed-package-contents
https://github.com/npm/pacote/issues/231

**Comment by wraithgar** [comment]
If I had to guess this is an existing bug in npm 6 that is now being raised because of a dependency that was previously hoisted now not being hoisted.  I don't know if it's a bug in any of these dependencies, the "breaking changes" were in the `engines` field only for the most part.

**Comment by raineorshine** [comment]
I have downgraded pacote to `15.0.0` until the `installed-package-contents` bug is fixed.

Try the latest npm-check-updates @ `v16.3.14`

**Comment by wraithgar** [comment]
https://github.com/npm/installed-package-contents/pull/26

**Comment by raineorshine** [comment]
Issue has been fixed in dependencies and dependency chain has been updated.

Published in `v16.3.20`.
